RetailCoder.VBE/Inspections/UntypedFunctionUsageInspectionResult.cs

```diff
@@ -16,7 +19,7 @@ public UntypedFunctionUsageInspectionResult(IInspection inspection, IdentifierRe
             _reference = reference;
             _quickFixes = new CodeInspectionQuickFix[]
             {
-                new UntypedFunctionUsageQuickFix(Context, QualifiedSelection), 
+                new UntypedFunctionUsageQuickFix((ParserRuleContext)GetFirst(typeof(VBAParser.IdentifierContext)).Parent, QualifiedSelection), 
```

Comment:
`GetFirst` could return `null`. If it's "not possible" for this to happen, it should be asserted. Otherwise a null check is in order.